client/src/components/Results/Results.css

```diff
@@ -253,6 +253,15 @@ on .card-phone-container*/
   justify-content: space-between;
 }
 
+.card-emergency{
```

Comment:
Nice work.  Other options are to change the background color of the emergency message with something like: 
```
.emergency-content {
  background-color: #ffe6e6;
}
```
or alter the emergency title with something like this: 
```
.emergency-item {
  background-color: #fc3c3c;
  padding: 4px;
  color: white;
}
```

Reply:
need more information on this change. My CSS is thrown off and I tried to delete card-emergency to get it back to normal but the css is off.

Reply:
I had an extra DIV and it off set the layout. NVM i'm going to play around with changing name to emergency-item.

Reply:
Do I make a new pull Request after making changes or is it the same PR I did yesterday?

Reply:
@isthewhiz I was just providing that css as examples your code.  For example that css could be attached to this: 
```
          {!(textMap.parsedEmergency === "") ? (
            <div className="card-item">
              <div className="card-title-flex">
                <div className="emergency-item">Emergency Message:</div>
              </div>
              <div className="card-content emergency-content">{textMap.parsedEmergency}</div>
            </div>
          ) : null}
```